node_modules/gh-events/tests/test-organisers.js

```diff
@@ -29,6 +30,46 @@ describe('Events', function() {
     describe('Organisers', function() {
 
         /**
+         * Test that verifies that organisers their personal information is hidden
+         */
+        it('verify organisers their personal information is hidden', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2013, 3, false, function(simon, nico, sam) {
+
+                var start = moment().format();
+                var end = moment().add(2, 'hour').format();
+
+                // Create an event
+                EventsTestUtil.assertCreateEvent(simon.client, 'Test event', start, end, null, function(event) {
+
+                    // Add Nico as an organiser
+                    var update = {};
+                    update[nico.profile.id] = true;
+                    EventsTestUtil.assertUpdateEventOrganisers(simon.client, event.id, update, function() {
+
+                        // Get the event
+                        EventsTestUtil.assertGetEvent(simon.client, event.id, null, function(event) {
+                            assert.strictEqual(event.organisers.length, 2);
+
+                            // Verify Nico's information is hidden
+                            var organisersById = _.indexBy(event.organisers, 'id');
+                            assert.strictEqual(organisersById[nico.profile.id].displayName, nico.profile.displayName);
```

Comment:
Maybe we can create a `assertPublicUser` function in the user tests util and call that to verify that the appropriate properties are present